account-kit/smart-contracts/src/ma-v2/account/account.ts

```diff
@@ -0,0 +1,150 @@
+import type {
+  EntryPointDef,
+  SmartAccountSigner,
+  SmartContractAccountWithSigner,
+  ToSmartContractAccountParams,
+} from "@aa-sdk/core";
+import {
+  createBundlerClient,
+  getAccountAddress,
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'getAccountAddress' is defined but never used.
<!-- __reviewdog__:ChBkMWUyMTlmYzJkNjJlYzM3EgZyZGpzb24= -->